### DISCUSSION TITLE
CVE-2024-6104: helmfile: hashicorp/go-retryablehttp: url might write sensitive information to log file

### DISCUSSION THREAD

**Discussion by manfred-h** [helmfile/helmfile#1597] [General]
Will there be a new release 0.166 with a fix for this CVE-2024-6104?

**Comment by yxxhero** [comment]
today

**Reply by manfred-h** [reply]
Thanks in advance!

**Comment by manfred-h** [comment]
Thanks a lot for the new release [v0.166.0](https://github.com/helmfile/helmfile/releases/tag/v0.166.0)!
